pkg/services/annotations/annotationstest/fake.go

```diff
@@ -0,0 +1,83 @@
+package annotationstest
```

Comment:
Nitpick: Does this need to be in a separate package? I usually see the fakes/mocks in the same package the interfaces are in. What do you think?

Reply:
I think that this was the convention; at least that's what we have done for the [preference](https://github.com/grafana/grafana/tree/main/pkg/services/preference/preftest) service.